src/mon/MonClient.cc

```diff
@@ -267,6 +268,11 @@ bool MonClient::ms_dispatch(Message *m)
   switch (m->get_type()) {
   case CEPH_MSG_MON_MAP:
     handle_monmap(static_cast<MMonMap*>(m));
+    if (passthrough_monmap) {
```

Comment:
As seen in the rest of the commit, this just makes it so the message gets shown to other Dispatchers — it enables some Mgr functionality that is otherwise dead.